index.js

```diff
@@ -259,14 +268,16 @@ module.exports = function(THREE) {
                 // restrict phi to be betwee EPS and PI-EPS
                 phi = Math.max( EPS, Math.min( Math.PI - EPS, phi ) );
 
+                scale += scaleDelta * (( this.enableDamping === true ) ? this.dampingFactor : 1);
```

Comment:
Are you talking about scale and scaleDelta?

Doesn’t matter if the radius has boundaries or not, we can’t zoom in/out further than the ‘desiredZoomScale’ we set.